assets/queries/cloudFormation/accessKey_not_rotated_within_90_days/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Access_Key_Is_Not_Rotated_Within_90_Days",
+  "queryName": "Access Key Is Not Rotated Within 90 Days",
```

Comment:
please change directory name to lowercase only: `accesskey_not_rotated_within_90_days`